<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
	background-image: url(:/images/resources/images/login-background.png);
	border-radius: 20px;
}

QFrame#main_frame{
	background-color: #23233B;
	border-radius: 20px;
	margin: 80px;
	padding-top: 20px;
	padding-left: 30px;
	padding-right: 30px;
}

QLabel{
	color: rgb(217, 214, 215);
	max-height: 10px;
}

QLineEdit {
	padding-left: 20px;
	min-height: 40px;
	border-radius: 15px;
	background-color: #434459;
	color: rgb(217, 214, 215);
}

QLineEdit::focus{
	border: 2px solid rgb(98, 131, 255);
}

QLineEdit::hover{
	border: 2px solid rgb(98, 196, 255);
}

QPushButton#login_button{
	color: rgb(217, 214, 215);
	min-height: 35px;
	border: 2px solid rgb(126, 128, 128);
	border-radius: 15px;
}

QPushButton#login_button::hover{
	border: 2px solid rgb(98, 196, 255);
}

QPushButton#minimize_button{
	background-color: rgb(255, 251, 0);
	border-radius: 7px;
}

QPushButton#close_button{
	background-color: rgb(255, 38, 0);
	border-radius: 7px;
}

QLabel#error_label {
	color: red;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="topbar_frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="minimize_button">
            <property name="maximumSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Minimizar</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="close_button">
            <property name="maximumSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Fechar</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="main_frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="username_label">
         <property name="text">
          <string>Usu√°rio</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="username_edit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="password_label">
         <property name="text">
          <string>Senha</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="password_edit">
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="login_button">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="error_label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
